drivers/SmartThings/zwave-thermostat/src/test/test_stelpro_ki_thermoestat.lua

```diff
@@ -69,16 +69,16 @@ test.register_coroutine_test(
   end
 )
 
-test.register_coroutine_test(
-  "Setting heating setpoint should not be handled",
-  function()
-    mock_device:set_field(constants.TEMPERATURE_SCALE, 1, {persist = true})
-
-    test.timer.__create_and_queue_test_time_advance_timer(1, "oneshot")
-    test.socket.capability:__queue_receive({ mock_device.id, { capability = "thermostatHeatingSetpoint", command = "setHeatingSetpoint", args = { 333.0 } } })
-  end
-)
+-- This test case should not be necessary as the bounds should be handled by the range in the profile
```

Comment:
Do you plan to eventually remove this or leave the comment here for posterity?

Reply:
open to either. what do you think?

Reply:
I'm fine with just removing the commented out test since it isn't needed anymore and the embedded configs are becoming more widespread so it's something I think someone would be familiar with when looking at the driver.